.travis.yml

```diff
@@ -34,7 +34,7 @@ matrix:
         - CMD="mvn install --projects jmh-tests,performance-tests --also-make --activate-profiles all -DskipTests=true --batch-mode --show-version --errors"
 
 script:
-- if [[ "$TRAVIS_EVENT_TYPE" == "cron" && DESC="unit tests" ]];
+  if [[ "$TRAVIS_EVENT_TYPE" == "cron" && "$DESC" == "unit tests" ]];
   then mvn install --batch-mode --show-version --errors sonar:sonar -Dsonar.host.url=https://sonarqube.com -Dsonar.login=${SONARQUBE_TOKEN} -Dsonar.organization=eclipse-collections ;
```

Comment:
@motlin not sure I understand your proposal regarding `$SONAR_COMMAND`, because I don't see a phase that is executed early enough to set variable before it is consumed in matrix - see little experiment at https://github.com/Godin/travis-playground/blob/e4859a718dd3d5eb09a7c045fcb5fa7b8dc9b61c/.travis.yml and respective build at https://travis-ci.org/Godin/travis-playground/builds/215665470

Also IMO far more readable and flexible will be to leave only `DESC` in `matrix` and do a switch on `DESC` in script. WDYT?